### PR TITLE
Add `translation/analyze-plain-text-sentiment` capability

### PR DESCRIPTION
**Pull request by janhalama**
## Description

This PR adds `translation/analyze-plain-text-sentiment` capability with `mock`, `google-apis-language`, `ibm-cloud-natural-language-understanding` providers and maps. 

## Motivation and Context
Extend toolbox capabilities.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
- [x] I haven't repeated the code. (DRY)
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: nock/translation/analyze-plain-text-sentiment/ibm-cloud-natural-language-understanding-eu-de/AnalyzePlainTextSentiment/recording.json [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+      "Tue, 05 Oct 2021 09:55:29 GMT",
+      "Connection",
+      "close"
+    ],
+    "reqheaders": {
+      "accept": [
+        "*/*"
+      ],
+      "authorization": [
+        "Basic YXBpa2V5OnJlZGFjdGVk"
```
Comment (zdne) [inline nock/translation/analyze-plain-text-sentiment/ibm-cloud-natural-language-understanding-eu-de/AnalyzePlainTextSentiment/recording.json:73]: @janhalama is this OK?

Comment (zdne) [reply nock/translation/analyze-plain-text-sentiment/ibm-cloud-natural-language-understanding-eu-de/AnalyzePlainTextSentiment/recording.json:73]: + it is repeated in other situations

File: providers/ibm-cloud-natural-language-understanding-eu-de.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "ibm-cloud-natural-language-understanding-eu-de",
+  "services": [
+    {
+      "id": "default",
+      "baseUrl": "https://api.eu-de.natural-language-understanding.watson.cloud.ibm.com"
```
Comment (zdne) [inline providers/ibm-cloud-natural-language-understanding-eu-de.json:6]: can we use the temporary workaround with the input variables (see the discussion in slack)?

File: capabilities/translation/analyze-plain-text-sentiment/maps/google-apis-language.suma [hunk 3]
```diff
@@ -0,0 +1,72 @@
+profile = "translation/analyze-plain-text-sentiment@1.0"
+provider = "google-apis-language"
+
+map AnalyzePlainTextSentiment {
+
+  http POST "/v1/documents:analyzeSentiment" {
```
Comment (zdne) [inline capabilities/translation/analyze-plain-text-sentiment/maps/google-apis-language.suma:6]: they really have urls with colon ? 🤯

File: capabilities/translation/analyze-plain-text-sentiment/maps/google-apis-language.suma [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        title = "Internal server Error"
+        detail = body.error.message
+      }
+    }
+  }
+}
+
+operation MapScoreToSentiment {
+  set {
+    result = (() => {
```
Comment (zdne) [inline capabilities/translation/analyze-plain-text-sentiment/maps/google-apis-language.suma:58]: This is OK but I wonder if we do not have a nicer way to do this common switch / mutliple ifs task e.g. using `set if`

**Review by zdne — CHANGES_REQUESTED** [review]
Overall looks good. Added some minor comments. One major concern is the basic hash being in the recording. Was this done with the latest testing library?

File: capabilities/translation/analyze-plain-text-sentiment/maps/google-apis-language.suma (hunk 4 above)
Comment (zdne) [reply capabilities/translation/analyze-plain-text-sentiment/maps/google-apis-language.suma:58]: @TheEdward162  what is the nicest way to do this? :D

**Comment by janhalama** [comment]
> One major concern is the basic hash being in the recording. Was this done with the latest testing library?

@zdne the hash contains redacted token value, so it should not be problem. I will look into latest testing library and use it.

File: capabilities/translation/analyze-plain-text-sentiment/maps/google-apis-language.suma (hunk 3 above)
Comment (janhalama) [reply capabilities/translation/analyze-plain-text-sentiment/maps/google-apis-language.suma:6]: yes see the docs: https://cloud.google.com/natural-language/docs/reference/rest/v1/documents/analyzeSentiment

File: providers/ibm-cloud-natural-language-understanding-eu-de.json (hunk 2 above)
Comment (zdne) [reply providers/ibm-cloud-natural-language-understanding-eu-de.json:6]: yea that looks good !

File: nock/translation/analyze-plain-text-sentiment/ibm-cloud-natural-language-understanding-eu-de/AnalyzePlainTextSentiment/recording.json (hunk 1 above)
Comment (janhalama) [reply nock/translation/analyze-plain-text-sentiment/ibm-cloud-natural-language-understanding-eu-de/AnalyzePlainTextSentiment/recording.json]: I updated testing-lib and was forced to update also one-sdk. Both in beta versions.

File: providers/ibm-cloud-natural-language-understanding-eu-de.json (hunk 2 above)
Comment (janhalama) [reply providers/ibm-cloud-natural-language-understanding-eu-de.json]: I removed region from provider name and injected it using input variables.

Comment (janhalama) [reply providers/ibm-cloud-natural-language-understanding-eu-de.json]: @zdne But I am not sure about the solution. Latest proposal from @smizell disallows using inputs in provider url definitions. See: https://www.notion.so/superface/Immutable-providers-a4d9b3b4af3f4635973315393b631407#f53f7f929d3647d19da01f51e3a38f52

**Comment by janhalama** [comment]
@freaz, @Jakub-Vacek  this PR has already been approved by "Z" but I had to rebase it and update recordings, please check it quickly

Linter is passing, tests as well.

**Review by martinalbert — APPROVED** [review]

**Pull request merged**